docs/interfaces/webui-if.rst

```diff
@@ -4,12 +4,93 @@
  Web Interface (WebUI)
 #######################
 
-The web interface, WebUI, provides a convenient way to create and monitor experiment progress.
+You can use the WebUI to create and monitor experiment progress.
 
-Access WebUI by visiting default URL ``http://master-addr:8080``, where ``master-addr`` is the
-hostname or IP address of the master. Use port ``8443`` if TLS is enabled.
+*****************
+ Getting Started
+*****************
 
-WebUI automatically redirects to the login page if a valid Determined session has not been
-established. After logging in, you are redirected to your initial URL.
+To access the WebUI, go to the default URL ``http://master-addr:8080`` in your web browser, where
+``master-addr`` is the hostname or IP address of the master. If Transport Layer Security (TLS) is
+enabled, use port ``8443`` instead.
 
-Click **Sign Out** at the top-right to end the Determined session.
+If you have not yet established a valid Determined session, the WebUI will automatically redirect
+you to the login page. Once you log in, you will be redirected to the initial URL you entered.
+
+To end the Determined session and log out of the WebUI, click **Sign Out**.
+
+************************
+ Creating an Experiment
+************************
+
+If you have an existing experiment or trial, you can use the WebUI to create an experiment. To find
+out more, visit :ref:`experiments`.
+
+**********************************************************
+ Organizing Your Experiments into Projects and Workspaces
+**********************************************************
+
+The WebUI lets you organize your experiments into projects and workspaces. To get started, visit
+:ref:`workspaces`.
+
+*******************************************
+ Configuring the Behavior of an Experiment
+*******************************************
+
+Many experiment configuration settings can be performed via the WebUI. To find out more, visit
```

Comment:
Same comment as above applies here and below (and to a lesser extent in "workspaces and projects"). Whichever route you like.

Reply:
i agree; changes made